hpeOneView/resources/servers/server_profile_templates.py

```diff
@@ -25,10 +25,10 @@
 
 from copy import deepcopy
 
-from hpeOneView.resources.resource import Resource, ensure_resource_client
+from hpeOneView.resources.resource import Resource, ResourcePatchMixin, ensure_resource_client
 
 
-class ServerProfileTemplate(Resource):
+class ServerProfileTemplate(Resource, ResourcePatchMixin):
```

Comment:
I cant see any other change expect ResourcePatchMixin being inherited. 
Is this being utilized, any reason to why have we inherited this?

Reply:
Yes, patch method is used in examples file which inherited from ResourcePatchMixin